README.md

```diff
@@ -38,6 +38,8 @@ To download a Kolibri WHL file, you can use `make whl=<URL>` from the command li
 
 6. Run `make kolibri.apk.unsigned` to build the apk. Watch for success at the end, or errors, which might indicate missing build dependencies or build errors. If successful, there should be an APK in the `dist/` directory.
 
+PS. If `p4a` command is not found, please check the ticket: ["p4a: command not found"](https://github.com/kivy/python-for-android/issues/1167)
```

Comment:
Instead of pointing to the issue, I think it would be better to document the problem:

> If you installed it with `--user`, make sure that `~/.local/bin` is in your `$PATH`.

I suppose we could make this more resilient by always invoking p4a with `python3 -m pythonforandroid`.

Reply:
Although, p4a tends to do a bunch of shenanigans with `sys.argv`, so that may not actually work.

Reply:
Yeah, I have the same thought - that issue is more like a support request ;) I definitely agree with explaining the possible error here, but we may as well say what to do in the same place.

Reply:
Sure!  Make sense.  And, updated.